### ISSUE TITLE
Ender Pouch doesnt open with hotkey | Potential incompatibility with Easy Shulker Boxes & Progression suggestion

### ISSUE THREAD

**Issue by hyperdotbat** [Draylar/inmis#164]
I presume this is because it simulates an actual Ender Chest
Also its weird how you can make a baby backpack and frayed backpacks but not upgrade the prior to the later, I supposed its easy to just throw in a recipe for that into the .jar file
[baby_to_frayed_backpack.json](https://github.com/Draylar/inmis/files/13954703/baby_to_frayed_backpack.json)
Also the same for upgrading Blazing To Wither
[blazing_to_withered_backpack.json](https://github.com/Draylar/inmis/files/13954750/blazing_to_withered_backpack.json)

Also would be cool if the progression switched to make a Copper backpack have 2 rows, then Iron 3 (instead of 2) and Gold 4 instead of 3 so its all in line with the diamond backpack having 5 rows

I suppose its possible to add new backpacks by tinkering with the config and other files
But cant seem to make it render the bundle-like behavior
![javaw_fgr3VIXBxP](https://github.com/Draylar/inmis/assets/48921794/c4c34c2d-5942-46ea-a4c8-62b8885191ee)

Ah and also modifying values is broken (maybe incompatibility with another mod, [Easy Shulker Boxes](https://modrinth.com/mod/easy-shulker-boxes) ?)
![javaw_1HqxtdYE1T](https://github.com/Draylar/inmis/assets/48921794/2000020e-a895-4a54-a15e-b00670dc9eb6)
After I took the Oak Log out, the bark just disappeared completely
I have the whole modified mod on mega: https://mega.nz/file/0G8ykAIA#DJ3giwJpqPTmG8m1TpgVv3NNfisEEffxS6LB_eV00Vc
And here's the config file:
[inmis.json](https://github.com/Draylar/inmis/files/13955179/inmis.json)

**Comment by hyperdotbat** [comment]
Ah I guess it was necessary to edit EasyShulkerBoxes to fix the issue with slots, not sure if its an inherent issue on their side or did it autogenerate those files (dont think so)
![image](https://github.com/Draylar/inmis/assets/48921794/c7c013f3-1b49-4276-9cb3-82c8b7a09654)
![javaw_J9InJPdnkR](https://github.com/Draylar/inmis/assets/48921794/ce95f0c1-d648-4b2c-b81c-6dd1d90fa5bc)
Here's also the updated version of that:
https://mega.nz/file/sHcgXQIT#exxFGOmILPmLfwjlUrPA_dd4MvIgw8_mPWY0LeQACow

**Comment by peow** [comment]
> Ah I guess it was necessary to edit EasyShulkerBoxes to fix the issue with slots, not sure if its an inherent issue on their side or did it autogenerate those files (dont think so) ![image](https://private-user-images.githubusercontent.com/48921794/297206809-c7c013f3-1b49-4276-9cb3-82c8b7a09654.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDcxNzk4ODMsIm5iZiI6MTcwNzE3OTU4MywicGF0aCI6Ii80ODkyMTc5NC8yOTcyMDY4MDktYzdjMDEzZjMtMWI0OS00Mjc2LTljYjMtODJjOGI3YTA5NjU0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMDYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjA2VDAwMzMwM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQyNjlkY2QyZjMyN2FiMDU4ZmI3NDBhNGFmOTVhMzVjZjcyN2Q4NzE4MGNkMDJmM2NmYjUyMWU5OWQxZGQxOGQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.jNSYzDdFBILYM-tKB-apr7u-XKy3ZHMUybOEHGTO22s) ![javaw_J9InJPdnkR](https://private-user-images.githubusercontent.com/48921794/297207440-ce95f0c1-d648-4b2c-b81c-6dd1d90fa5bc.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDcxNzk4ODMsIm5iZiI6MTcwNzE3OTU4MywicGF0aCI6Ii80ODkyMTc5NC8yOTcyMDc0NDAtY2U5NWYwYzEtZDY0OC00YjJjLWI4MWMtNmRkMWQ5MGZhNWJjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMDYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjA2VDAwMzMwM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTViNzhiMjFjZTE5ZmNkZmE2MDE5MGNhYjFmYzkzNjg1YjY0OGI1MmIwMTJhNGNiMTRjNGFmMjFmZTgxY2U0M2MmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.NP4eJMzuKjyhE1yqI1_YaqMWdZs1NOPWaOqxCj-udd0) Here's also the updated version of that: https://mega.nz/file/sHcgXQIT#exxFGOmILPmLfwjlUrPA_dd4MvIgw8_mPWY0LeQACow

I believe these Easy Shulker Boxes files are to prevent you from putting one backpack inside another, right?

----------------

about you cannot upgrade Blazing To Withered backpack, I believe this is on purpose because the Blazing Backpack is immune to lava, a great feature, unlike the Withered Backpack, which on the other hand has greater storage and also can be upgraded to a even bigger storage

**Comment by hyperdotbat** [comment]
> about you cannot upgrade Blazing To Withered backpack, I believe this is on purpose because the Blazing Backpack is immune to lava, a great feature, unlike the Withered Backpack, which on the other hand has greater storage and also can be upgraded to a even bigger storage

yeah I get that, realized that later, but still there could be a recipe to just upgrade from Blazing to Withered if you want to
And definitely a one to upgrade from Baby to Frayed
